lib/anyway_config.rb

```diff
@@ -35,21 +35,17 @@ def loaders
     end
   end
 
-  def self.register_loaders
-    # Configure default loaders
-    loaders.append :yml, Loaders::YAML
-    loaders.append :env, Loaders::Env
-
-    # Configure optional loaders
-    if Utils.which("ejson")
-      loaders.append :ejson, Loaders::EJSON
-    end
-    if ENV.key?("DOPPLER_TOKEN") && ENV["ANYWAY_CONFIG_DISABLE_DOPPLER"] != "true"
-      loaders.append :doppler, Loaders::Doppler
-    end
-  end
+  # Configure default loaders
+  loaders.append :yml, Loaders::YAML
+  loaders.append :env, Loaders::Env
 
-  register_loaders
+  # Configure optional loaders
+  if Utils.which("ejson")
```

Comment:
I think, it worths adding `ENV["ANYWAY_CONFIG_DISABLE_EJSON"]`, so we can skip the `which` call, too